codegen/module.go

```diff
@@ -269,6 +269,54 @@ func (system *ModuleSystem) ResolveModules(
 					append(resolvedDependencies, dependencyInstance)
 			}
 		}
+
+		// Resolve the class dependencies
+		for _, classInstance := range classInstances {
+			for _, classDependency := range classInstance.Dependencies {
+				moduleClassInstances, ok :=
+					resolvedModules[classDependency.ClassName]
+
+				if !ok {
+					return nil, errors.Errorf(
+						"Invalid class name \"%q\" in dependencies for %s %s",
```

Comment:
I am guessing `"Invalid class name %q in dependencies for %q %q"` is what you want here?